bin/extract_to_arb.dart

```diff
@@ -20,64 +18,84 @@ import 'package:intl_translation/src/directory_utils.dart';
 import 'package:path/path.dart' as path;
 
 void main(List<String> args) {
-  String targetDir;
-  String outputFilename;
-  String sourcesListFile;
-  bool transformer;
-  var parser = ArgParser();
-  var extraction = MessageExtraction();
-  String locale;
+  String targetDir = '.';
```

Comment:
Are these top-level variables which are allowed to keep their type, or should I also change the type to `var` here?

Reply:
This is just a regular local variable. A top-level variable would be one declared in library scope; there aren't any examples of top-level variables in this library, but `bin/rewrite_intl_messages.dart` has three (`outputFileOption`, ...).